tests/phpunit/tests/rest-api/rest-global-styles-controller.php

```diff
@@ -130,6 +119,89 @@ public function test_context_param() {
 		// Controller does not use get_context_param().
 	}
 
+	public function test_get_theme_items() {
```

Comment:
There seem to be a lot more things in this test than there are in the one in Gutenberg - the expected array is a lot bigger. I assume that's because we're testing with a different theme. I think we should think about using the same test theme in both places, but in a different PR :)

Reply:
> I assume that's because we're testing with a different theme.

Yep.

> I think we should think about using the same test theme in both places, but in a different PR :)

👍